waspc/data/Cli/templates/skeleton/tsconfig.json

```diff
@@ -5,6 +5,10 @@
 // compiler. Proper TS compiler configuration in Wasp is coming soon :)
 {
   "compilerOptions": {
+    "target": "esnext",
+    // We're bundling all code in the end so this is the most appropriate option,
+    // it's also important for autocomplete to work properly.
+    "moduleResolution": "bundler",
```

Comment:
I got here by trial and error, but `bundler` seems the most (if not only) appropriate option for our setup.

By implicit defaults cascading, this got set `CommonJs`, which is why it failed to recognize ES exports from the `wasp` package.

The cascading was:
- `target` was unset and defaulted to `ES3` 
- `module` was unset and defaulted to `CommonJS` because the `target` was `ES3`
- `moduleResolution` was unset and defaulted to `Node` because `module` was `CommonJS`

Source for all rules: https://www.typescriptlang.org/tsconfig#module

Reply:
Given that this is in the user's `tsconfig.json`, does this mean that only the new newly created projects will get the benefits of these changes?

Do we also need to update _all_ other templates?

Reply:
> Given that this is in the user's tsconfig.json, does this mean that only the new newly created projects will get the benefits of these changes?

Yes, any ideas on what we can do here?

> Do we also need to update all other templates? Specifically the ones you get when running wasp new:

Apparently yes, nice catch! I thought they all used `skeleton`... We really do need to change those names :smile:

Reply:
Yeah, those names are a bit tricky. But don't they use `skeleton`? Except for open-saas?

Reply:
I tested it out, and unfortunately not.

Reply:
Btw, what do I do with that?

Should I do anything now or do we take care of it in the release process: https://github.com/wasp-lang/wasp/tree/main/waspc#typical-release-process.

Reply:
I think it makes sense to do now, when we still have context about it and it is fresh.

But you can do it another PR if you wish! Or maybe you even have to, for these starters.